File: Libraries/rust/beagle_helper/src/boards/pocketbeagle2.rs
use crate::pin::{GpioPin, Pin, PwmPin};

pub const P1_20: Pin = Pin::new(Some(GpioPin::new(2, 50)), None);
pub const P1_36: Pin = Pin::new(None, Some(PwmPin::new(0, 0)));

pub const P2_30: Pin = Pin::new(None, Some(PwmPin::new(2, 0)));
pub const P2_33: Pin = Pin::new(Some(GpioPin::new(2, 52)), None);

pub mod techlab {
    use crate::pin::Pin;

    pub const BUZZER: Pin = super::P2_30;

    pub const LED: &str = "/sys/devices/platform/techlab-led/leds/multi-led/";
}

File: Libraries/python/beagle_helper/src/beagle_helper/boards/pocketbeagle2.py
from pathlib import Path
from ..pin import Pin, GpioPin, PwmPin

P1_20 = Pin(gpio=GpioPin("2", 50))
P1_36 = Pin(pwm=PwmPin(0, 0))

P2_30 = Pin(pwm=PwmPin(2, 0))
P2_33 = Pin(gpio=GpioPin("2", 52))

EEPROM_PATH = "/sys/bus/i2c/devices/0-0050/eeprom"

TECHLAB_LED = Path("/sys/devices/platform/techlab-led/leds/multi-led/")
